facebook/Module.md

```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
-}
 
 facebook:FacebookConfiguration facebookConfig = {
     clientConfig:{
         auth:{
             scheme: http:OAUTH2,
-            accessToken:accessToken
+            accessToken: config:getAsString("ACCESS_TOKEN")
         }
     }
 };
 
 facebook:Client facebookclient = new(facebookConfig);
 
-function callMethodsWithUserToken(string userAccessToken) returns string {
-    
+string pageAccessToken = "";
+public function main(string... args) {
+    pageAccessToken = untaint callMethodsWithUserToken();
+    callMethodsWithPageToken();
+}
+
+function callMethodsWithUserToken() returns string {
     io:println("-----------------Calling to get page accessTokens details------------------");
     //Get page tokens
-    var tokenResponse = client->getPageAccessTokens("me");
-    //facebook:AccessTokens accessTokenList = {};
-    string pageToken;
+    var tokenResponse = facebookclient->getPageAccessTokens("me");
+    string pageToken = "";
 
     if (tokenResponse is facebook:AccessTokens) {
-            pageToken = tokenResponse.data[0].pageAccessToken;
-            io:println("Page token details: ");
-            io:println(tokenResponse.data);
-            io:println("Page Name: ");
-            io:println(tokenResponse.data[0].pageName);
-            io:println("Page token: ");
-            io:println(pageToken);
+        pageToken = tokenResponse.data[0].pageAccessToken;
+        io:println("Page token details: ", tokenResponse.data);
+        io:println("Page Name: ", tokenResponse.data[0].pageName);
+        io:println("Page token: ", pageToken);
     } else {
-        io:println(tokenResponse.detail().message);
+        io:println("Error: ", tokenResponse.detail().message);
     }
 
     io:println("-----------------Calling to get friends list details------------------");
     //Get Friends list details
-    var friendsResponse = client->getFriendListDetails("me");
+    var friendsResponse = facebookclient->getFriendListDetails("me");
     if (friendsResponse is facebook:FriendList) {
-        io:println("Friends list: ");
-        io:println(friendsResponse.data);
-        io:println("Friends list count: ");
-        io:println(list.summary.totalCount);
+        io:println("Friends list Details: ", friendsResponse.data);
+        io:println("Friends list count: ", friendsResponse.summary.totalCount);
     } else {
-        io:println(e.message);
+        io:println("Error: ", friendsResponse.detail().message);
     }
     return pageToken;
 }
 
-function callMethodsWithPageToken(string pageAccessToken) {
-    endpoint facebook:Client client {
-        clientConfig:{
-            auth:{
-                accessToken:pageAccessToken
-            }
+facebook:FacebookConfiguration facebookPageConfig = {
+    clientConfig:{
+        auth:{
+            scheme: http:OAUTH2,
+            accessToken: getpageToken()
         }
-    };
+    }
+};
+
+facebook:Client facebookPageclient = new(facebookPageConfig);
```

Comment:
Better to rename `facebookPageclient` -> `facebookPageClient` including all the usages.